Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -364,29 +374,38 @@ def start(self):
         self.log_observer = ParseByLineLogObserver(self.parseOutputLine)
         self.addLogObserver('stdio', self.log_observer)
 
+        command = [part.getRenderingFor(self.build) if isinstance(part, WithProperties) else part for part in self.command]
```

Comment:
Is this needed?

Reply:
We can technically use `getProperty` instead, but this is consistent with how EWS does it.

Reply:
If we can use getProperty, let's use that (and we can change it in EWS as well). That would be more maintainable.